src/psyclone/psyir/symbols/datatypes.py

```diff
@@ -48,6 +48,7 @@
 from psyclone.psyir.symbols.data_type_symbol import DataTypeSymbol
 from psyclone.psyir.symbols.datasymbol import DataSymbol
 from psyclone.psyir.symbols.symbol import Symbol
+from psyclone.psyir.commentable_mixin import CommentableMixin
```

Comment:
We have the rule that these should be order alphabetically, so move this line before the `from psyclone.psyir.symbols ...`

Reply:
Done